cmd/goci/changes.go

```diff
@@ -53,8 +53,20 @@ func actionChanges() error {
 			}
 		}
 
-		fmt.Printf("%s\n", strings.Join(res, " "))
+		fmt.Printf("%s\n", strings.Join(filterUnique(res), " "))
 	}
 
 	return nil
 }
+
+func filterUnique(changes []string) []string {
```

Comment:
Please integrate these changes into the previously added code for unique element checking(e.g. line 44)